section4/exercises/burrito.rb

```diff
@@ -11,9 +11,29 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(new_topping)
+    toppings.push(new_topping)
+  end
+
+  def remove_topping(old_topping)
+    toppings.delete(old_topping)
+  end
+
+  def change_protein(new_protein)
+    @protein = new_protein
```

Comment:
Nice work with your dynamic methods that take params! Also, you used the `@` correctly here- you'll get more familiar with when to use this, but in general, it is needed with many class attributes.